src/aero_kernel/src/syscall/mod.rs

```diff
@@ -210,6 +210,7 @@ extern "C" fn __inner_syscall(sys: &mut SyscallFrame, stack: &mut RegistersFrame
         SYS_UNLINK => fs::unlink(b, c, d, e),
 
         SYS_SOCKET => net::socket(b, c, d),
+        SYS_BIND => net::bind(b, unsafe { &*(c as *const SocketAddr) }, d),
```

Comment:
To keep everything consistent, maybe cast `c` to `SocketAddr` in the bind function itself.

Reply:
Fixed in f02d8bc3e31134c1ed0cfbcae21b3f57c321a1b1